content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -129,6 +129,51 @@ the [kubeadm config migrate](/docs/reference/setup-tools/kubeadm/kubeadm-config/
 For more information on the fields and usage of the configuration you can navigate to our
 [API reference page](/docs/reference/config-api/kubeadm-config.v1beta3/).
 
+### Using kubeadm init with feature gates {#feature-gates}
+
+Kubeadm support a set of feature gates that are unique to kubeadm and can only be applied
+during cluster creation with `kubeadm init`. These features can control the behavior
+of the cluster. Feature gates are removed after a feature graduates to GA.
+
+ To pass a feature gate you can either use the `--feature-gates` flag for
+`kubeadm init` or use the `ClusterConfiguration.featureGates` field when you pass
+a configuration file using `--config`.
+
+Passing [feature gates for core Kubernetes components](/docs/reference/command-line-tools-reference/feature-gates)
+directly to kubeadm is not supported. Instead, it is possible to pass them by
+[Customizing components with the kubeadm API](/docs/setup/production-environment/tools/kubeadm/control-plane-flags/).
+
+List of feature gates:
+
+```
+| Feature | Default | Stage | Since |
+|---------|---------|-------|-------|
+| `PublicKeysECDSA` | `false` | Alpha | 1.19 |
+| `RootlessControlPlane` | `false` | Alpha | 1.22 |
+| `UnversionedKubeletConfigMap` | `true` | Beta | 1.23 |
+```
+
+Feature gate descriptions:
+
+- `PublicKeysECDSA`: Can be used to create a cluster with ECDSA certificates instead of the
+default RSA algorithm. Renewal of existing ECDSA certificates is also supported using
+`kubeadm certs renew`, but not switching between the RSA and ECDSA algorithms on the fly
+or during upgrades.
+
+- `RootlessControlPlane`: Can be used to enable the kubeadm deployed control plane component
+static Pod containers for `kube-apiserver`, `kube-controller-manager`, `kube-scheduler`
+and `etcd` to run as a non-root users. Switching the feature gate to enabled or disabled
+is possible before upgrade.
+
+- `UnversionedKubeletConfigMap`: When this feature is enabled kubeadm will start using a new naming
+format for the ConfigMap where it stores the `KubeletConfiguration` structure. The old format includes
+the Kubernetes version (for example, `kube-system/kubelet-config-1.22`), while the new format does not
+(`kube-system/kubelet-config`). A similar formatting change is done for the related RBAC rules.
+The old format is DEPRECATED and will be removed after the feature graduates to GA. When writing
+the ConfigMap (during `kubeadm init`, `kubeadm upgrade apply`) kubeadm will respect the value
+of `UnversionedKubeletConfigMap`, while when reading it (during `kubeadm join`, `kubeadm reset`,
+`kubeadm upgrade ...`), it would attempt to use new format first and fallback to the legacy format if needed.
```

Comment:
thanks for the edits. updated to use these, with one minor diff:
`kubeadm tool -> kubeadm` since kubeadm is the project name and is considered more of a toolkit

> I'd use a definition list

i have not used formatting such as:

```
`foo`
: text
```

but i guess i can check how that looks in the preview.
updated.

Reply:
[preview](https://deploy-preview-31687--kubernetes-io-vnext-staging.netlify.app/docs/reference/setup-tools/kubeadm/kubeadm-init/#feature-gates)